test/Parse/ConditionalCompilation/armAndroidTarget.swift

```diff
@@ -7,7 +7,7 @@
 let i: Int = "Hello"
 #endif
 
-#if arch(arm) && os(Android) && _runtime(_Native) && _endian(little)
+#if arch(arm) && os(Android) && _runtime(_Native) && _endian(little) && _pointerBitWidth(_32)
```

Comment:
This seems pointless if it's not possible to use other pointer sizes on platforms like Android armv7.

Reply:
I'm not sure I understand. In what cases would you want other pointer sizes on armv7?

Reply:
My point is that isn't `arch(arm)` here redundant with `_pointerBitWidth(_32)`, ie the former always implies the latter?

Reply:
Oh, but the whole point of this test is to check that all of these are properly set. If any of them wasn't, the test would fail.

Reply:
OK, so the goal is to just lump together all compile-time properties of this platform, even though some of them are redundant so nobody would ever use some combos of those checks together, got it.

Reply:
Not quite. It is to ensure that if we are compiling for `armv7-unknown-linux-androideabi`, all of the expected conditions hold. We have these tests for all targets.